src/encoded/static/components/item-pages/HiGlassViewConfigView.js

```diff
@@ -1095,6 +1128,10 @@ function StatusMenuItem(props){
  */
 export const ConfirmModal = React.memo(function (props) {
     const { handleConfirm, handleCancel, modalTitle, confirmButtonText = "OK", cancelButtonText = "Cancel" } = props;
+    let confirmButton = null;
+    useEffect(() => {
+        confirmButton.focus();
+    });
```

Comment:
Not sure if the following would be beneficial or not but we could pass an empty array as second parameter which would make this run only once on mount rather than every time is rendered (from ["Tip: Optimizing Performance by Skipping Effects"](https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects)).

Reply:
I have noticed you already pass an empty array while rebuilding tooltips via `useEffect`. https://github.com/4dn-dcic/fourfront/blob/53a338a00a9a3776bbcb7924b550a49e2df83c24/src/encoded/static/components/item-pages/HiGlassViewConfigView.js#L957

I tested with and without passing an empty array; at first glance, it does not add an extra benefit in this case. I added it though, keeping it that way prevents an unnecessary rendering in the future.